src/test/java/SetTest.java

```diff
@@ -0,0 +1,57 @@
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+@DisplayName("Set Collection에 대한 학습 테스트")
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Nested
+    @DisplayName("요구사항 1")
+    class Size_테스트 {
+        @Test
+        @DisplayName("Set의 size() 메소드를 활용해 Set의 크기를 확인하는 학습테스트를 구현한다.")
+        void Size_테스트() {
+            assertThat(numbers.size()).isEqualTo(3);
```

Comment:
```suggestion
            assertThat(numbers).hasSize(3);
```

JUnit5의 API를 활용해볼 수 있을 것 같아요!

Reply:
앗, 이 부분은 제가 놓쳤네요! 감사합니다 👍